samples/cellular/modem_shell/sample.yaml

```
@@ -92,14 +92,30 @@ tests:
       - nrf9161dk_nrf9161_ns
     platform_allow: nrf9160dk_nrf9160_ns nrf9161dk_nrf9161_ns
     tags: ci_build
-  sample.cellular.modem_shell.fota:
+  sample.cellular.modem_shell.app_fota:
     build_only: true
     extra_args: OVERLAY_CONFIG=overlay-app_fota.conf
     integration_platforms:
       - nrf9160dk_nrf9160_ns
       - nrf9161dk_nrf9161_ns
     platform_allow: nrf9160dk_nrf9160_ns nrf9161dk_nrf9161_ns
     tags: ci_build
+  sample.cellular.modem_shell.modem_fota_full:
```

```suggestion
  sample.cellular.modem_shell.modem_fota_full.nrf9160:
```

To align with 9161. I know there's similar issue elsewhere in this file and I think you could change that too.